api/src/accounts.rs

```diff
@@ -46,6 +48,7 @@ pub fn get_account_resources_by_ledger_version(
         .map(|version, address, ctx| (Some(version), address, ctx))
         .untuple_one()
         .and_then(handle_get_account_resources)
+        .with(metrics("get_account_resources_by_version"))
```

Comment:
Similarly, I'd keep the metrics and function name the same if you can for ease of looking up later.